nodejs/packages/layer/src/wrapper.ts

```diff
@@ -88,10 +87,19 @@ declare global {
   // No explicit metric type here, but "unknown" type.
   // Because metric packages are important dynamically.
   function configureMeter(defaultConfig: unknown): unknown;
+  /**
+   * @deprecated please use {@link configureMeter} instead.
+   */
   function configureMeterProvider(meterProvider: unknown): void;
 
+  // No explicit logger type here, but "unknown" type.
+  // Because logger packages are important dynamically.
+  function configureLogger(defaultConfig: unknown): unknown;
```

Comment:
Can we not make the return type `LoggerProviderConfig`? The `configureLogger` result is always used to instantiate  the `LoggerProvider`
```suggestion
  function configureLogger(defaultConfig: unknown): LoggerProviderConfig;
```

Reply:
@wpessers The reason is why I have used `unknown` type (instead of actual `LoggerProviderConfig` type) is preventing eager loading of the logger related modules in the webpack bundle (for ex. when `OTEL_LOGS_EXPORTER` env var is `none`) to reduce coldstart delay.

Reply:
Ah that makes sense. In that case the PR looks good to me.